app/src/main/java/com/github/onlynotesswent/model/note/Note.kt

```diff
@@ -44,11 +44,11 @@ data class Note(
     }
   }
 
-  enum class Type {
-    JPEG,
-    PNG,
-    PDF,
-    NORMAL_TEXT
+  enum class Type(val fileExtension: String) {
```

Comment:
Mybe called that NoteType, Type is a bit to vague

Reply:
For now it is a nested class inside Note, so it is technically Note.Type if only Note is imported. Either way this should be changed in a future PR, I decided to separate this from this PR because refactoring will cause many merge conflicts (I tried).